cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -585,7 +585,16 @@
     // NOTE: "v2" to avoid conflict with data of older metrics system
     public static final String DEFAULT_ENTITY_TABLE_NAME = "metrics.v2.entity";
     public static final String DEFAULT_METRIC_TABLE_PREFIX = "metrics.v2.table";
+
+    // NOTE: "v3" to avoid conflict with data of older metrics system
+    public static final String DEFAULT_METRIC_V3_TABLE_PREFIX = "metrics.v3.table";
+    public static final String METRICS_HBASE_MAX_SCAN_THREADS = "metrics.hbase.max.scan.threads";
+    public static final String METICS_HBASE_TABLE_SPLITS = "metrics.v3.table.splits";
+    // Hardcode this value because we do not want user to control it
```

Comment:
I dont think it should be in the dataset properties though. Only in the cconf, imo.